src/content/components/Triage/Triage.js

```diff
@@ -133,9 +138,49 @@ export class Triage extends React.PureComponent {
     return {};
   }
 
+  getSubHeading() {
+    let { triageOwner } = this.state;
+    if (!triageOwner) {
+      return null;
+    }
+    let { real_name, nick, email } = triageOwner;
+    let startDate = DateTime.utc()
+      .set({
+        weekday: 1,
+        hour: 12,
+        minute: 30,
+        second: 0,
+        millisecond: 0,
+      })
+      .toLocal();
+    let endDate = startDate.plus({ days: 4 });
+    let dateString = startDate.toLocaleString({
+      month: "numeric",
+      day: "numeric",
+    });
+    if (startDate.month !== endDate.month) {
+      dateString += ` - ${endDate.toLocaleString({
```

Comment:
This works great, though personally I would change the month format to "long" so that the date appears `(April 22-26)`; because right now it kinda looks like you intend to have triage until April 22nd, 2026 (04-22-26)

Reply:
LMAO good catch. No idea how I overlooked that. My brain has no excuse